### PR TITLE
Integrate Ryan's server updates

### PR DESCRIPTION
**Pull request by tdooner**
(Built atop #32)

@nihonjinrxs - I took a stab at rebasing your structure updates. Nice work! Everything seems to work locally, although I'm sure there will be another change or two necessary to alert Heroku/Netlify of the new structure.

There was one loose end I noticed:
- [ ] The mocking you did [in this test](https://github.com/codeforamerica/brigade-project-index-statusboard/pull/25/files#diff-f1fe78944ce096916d2f696a4dfda0b5R16-R33) didn't seem to work for me. (It actually loads the data, and the test fails.)

I'm not familiar with mocking `require`s (or much of jest TBH) so if you could take a quick look and LMK what to do about it that would be great

### CODE REVIEW

**Comment by nihonjinrxs** [comment]
Thanks so much for this work! Sorry I haven't been able to get back to this in a bit.

I can take a look at the mocking issue -- my memory is a sieve, but I had in mind that this PR wasn't quite ready and this is probably what caused that. I'll try to do this sometime this week.

**Comment by tdooner** [comment]
I wouldn't worry too much about it -- it looks like that test might not be getting us very much anyway. But if you have a chance, would love to learn about how to do what you were trying to do there :)

File: api/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "extends": ["airbnb", "plugin:prettier/recommended"],
```
Comment (mmazanec22) [inline api/.eslintrc.json:2]: Yay

File: statusboard/package.json [hunk 2]
```diff
@@ -11,12 +11,14 @@
     "d3-selection": "^1.4.1",
```
Comment (mmazanec22) [inline statusboard/package.json:11]: Note to self that we can get rid of these d3 libraries now

**Review by mmazanec22 — APPROVED** [review]
Yay thanks for doing this!

**Pull request merged**